docs/installation.md

```diff
@@ -97,27 +97,24 @@ sudo apt-get install build-essential git
 
 #### Raspberry Pi / Raspbian
```

Comment:
This should be moved 1 level up (so be with `###`)

It's a different case, not a part of 'normal' Linux installation instruction.